src/InfluxDbBundle.php

```diff
@@ -5,8 +5,16 @@
 use Algatux\InfluxDbBundle\DependencyInjection\InfluxDbExtension;
 use Symfony\Component\HttpKernel\Bundle\Bundle;
 
+/**
+ * Class InfluxDbBundle.
```

Comment:
* Class comment not needed
* a `trigger_error` should be added outside the class.